### ISSUE TITLE
Seemingly random freezing on "Dis"

### ISSUE THREAD

**Issue by rfmerrill** [viciious/d32xr#86]
Observed this both on my old build (from end of November) and on a fresh build I just made today.

I played perfectly fine all the way through to the last level but while fighting the Spider Mastermind I had the game hard-lock on me multiple times.

Finally I tried re-beating the previous level, and when I got to Dis I switched the SFX driver to PWM and I was able to complete it--though I did have a brief lockup where some text appeared at the top right but disappeared before I could read it. The game recovered however.

Hardware:
- VA0 US model 2 genesis with discrete 3438 and region switch mods (region set to US)
- US 32X
- US model 2 sega CD with region hacked Japanese bios.
- Mega Everdrive Pro
- PS4 controller with Brook converter

I know my hardware setup is weird so if this doesn't reproduce on more normal hardware that is totally fine.

**Comment by rfmerrill** [comment]
It also reproduces on my less "weird" tower (Japanese 32X, Japanese MCD1, Japanese VA6 MD1) and with the SFX engine set to PWM. 

I tried to see if I could capture the text but I couldn't, however in this video you can see one brief lockup (around 55 seconds) and then later a total hardlock: https://drive.google.com/file/d/1aBwWi6Y8TObHkb4UDzZ_QcuKr_uY8v8v/view?usp=sharing

**Comment by viciious** [comment]
Well it could be anything really.. From random EM interference affecting the flaky cables to ROM bank switching woes, which are specific to a particular MED Pro revision. How am I supposed to be able to debug and fix this?

**Comment by rfmerrill** [comment]
Probably can't with just the info I have. Maybe if someone else has the same issue they can comment more information here and we could narrow down what config causes it and that might point to some reproduction steps.

**Comment by rfmerrill** [comment]
I wasn't able to reproduce it on other model 2 genesis, so I replaced the cart slot on the offending one.

That didn't fix it, but then I realized this was one of the boards where I had removed the capacitor array as it caused instability when using the Mega Everdrive Pro. I guess with the 32X and Sega CD installed, the _lack_ of that capacitor bank was causing instability.

I found it, put it back on, seems stable now. Sorry to bother you about it--maybe 5 years from now someone does the same thing and finds this issue haha.
